packages/test/test-end-to-end-tests/src/test/gc/gcUnknownHandles.spec.ts

```diff
@@ -78,7 +78,11 @@ describeCompat("GC unknown handles", "FullCompat", (getTestObjectProvider) => {
 	 */
 	async function getGCNodesFromSummary() {
 		await provider.ensureSynchronized();
-		const { summary } = await summarizerRuntime.summarize({ runGC: true, trackState: false });
+		const { summary } = await summarizerRuntime.summarize({
+			runGC: true,
+			trackState: false,
```

Comment:
Remove trackState?

Reply:
I will do that in the other PR which removes trackState.